documentcloud/documents/models.py

```diff
@@ -379,6 +380,59 @@ def get_all_page_text(self):
             )
             return {"pages": [], "updated": None}
 
+    def set_page_text(self, page_text_infos):
```

Comment:
If the page is not edited, is the text stored in the database instead of uploaded to a bucket?

Reply:
Not sure exactly what you mean, but the text is never stored in the database

Reply:
Oh, OK. We can talk about it live, but if it's always stored on S3, I'm confused about why calls to `async_upload` are new. It seems like they should have been there before this PR.